src/components/Admin/Tag.jsx

```diff
@@ -40,6 +40,22 @@ const colors = {
     background: "bg-hackathon-judge-student-bg",
     text: "text-hackathon-judge-student-text",
   },
+  marketing: {
```

Comment:
make these color based not role based

Reply:
not sure if this is what you meant, but i changed the admin.jsx tag to use the judges colors instead.